packages/bpk-component-rating/src/BpkRating.scss

```diff
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins';
+
+.bpk-rating {
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: max-content auto;
+  grid-auto-flow: column;
+  grid-column-gap: $bpk-spacing-sm;
+
+  &--base-rating {
+    width: $bpk-text-base-line-height * 2;
+    height: $bpk-text-base-line-height * 2;
+    padding: $bpk-spacing-sm;
+    grid-column: 1;
+    grid-row: 1 / 3;
+    border-radius: $bpk-text-xl-line-height;
+    color: $bpk-color-white;
+    text-align: center;
+    vertical-align: middle;
+
+    @include bpk-themeable-property( background-color, --bpk-rating-green, $bpk-color-green-600 );
```

Comment:
Can we name the theme props 'high' 'medium' and 'low', so that they aren't tied to specific colours? Imagine if in future we made the high rating blue by default, now the variable name would be weird.

Reply:
Updated the colours as per the suggestion so will need to add it to definitions and also to themeAttributes